nemo_skills/pipeline/eval.py

```diff
@@ -226,6 +226,7 @@ def eval(
         cluster_config,
         log_dir=log_dir,
         mount_map={output_dir: None},
+        create_remote_dir=True,
```

Comment:
this is actually unnecessary, by the way. The inference/generate.py will always create this folder for you, so just an extra ssh access here that we can avoid. Not a big deal, so feel free to keep if you'd like

Reply:
I can drop it here.